step/step.go

```diff
@@ -252,3 +262,20 @@ func moveAssets(xcresultPath string, htmlReportDir string) error {
 
 	return nil
 }
+
+func createReportInfo(htmlReportDir string) error {
+	reportInfo := ReportInfo{
+		Category: "test",
+	}
+
+	jsonData, err := json.Marshal(reportInfo)
+	if err != nil {
+		return err
+	}
+
+	if err := os.WriteFile(filepath.Join(htmlReportDir, htmlReportInfoFile), jsonData, 0644); err != nil {
```

Comment:
Note: In the generateTestReport function, the test report dir path is created with permission: `0755`. By looking around the steps, we use: `os.ModePerm`, `0755`, and `0777`.
Was the current permission here intentional?

Reply:
It was not. I have changed it to `0755` to make it uniform in the step.